SU2_CFD/include/variables/CVariable.hpp

```diff
@@ -1647,6 +1666,46 @@ class CVariable {
    */
   inline virtual su2double GetCrossDiff(unsigned long iPoint) const { return 0.0; }
 
+  /*!
+   * \brief Get the value of the value of FTrans.
+   */
+  inline virtual su2double GetFTrans(unsigned long iPoint) const { return 0.0; }
+
+  /*!
+   * \brief Set the value of the value of FTrans.
+   */
+  inline virtual void SetFTrans(unsigned long iPoint, su2double val_FTrans) {}
+
+  /*!
+   * \brief Get the value of the value of FTrans.
+   */
+  inline virtual su2double GetVelLapl_X(unsigned long iPoint) const { return 0.0; }
+  /*!
+   * \brief Get the value of the value of FTrans.
+   */
+  inline virtual su2double GetVelLapl_Y(unsigned long iPoint) const { return 0.0; }
+  /*!
+   * \brief Get the value of the value of FTrans.
+   */
+  inline virtual su2double GetVelLapl_Z(unsigned long iPoint) const { return 0.0; }
+
+  /*!
+   * \brief Set the value of the value of FTrans.
+   */
+  inline virtual void AddVelLapl(unsigned long iPoint, su2double val_VelLapl_X, su2double val_VelLapl_Y) {}
+  /*!
+   * \brief Set the value of the value of FTrans.
+   */
+  inline virtual void AddVelLapl_Z(unsigned long iPoint, su2double val_VelLapl_Z) {}
+  /*!
+   * \brief Set the value of the value of FTrans.
+   */
+  inline virtual void SetVelLapl(unsigned long iPoint, su2double val_VelLapl_X, su2double val_VelLapl_Y) {}
```

Comment:
Use a pointer for velocity and then you only need one of each function instead of handling xyz separately.

Reply:
Yeah, I'll try using the matrix type